app/routes/_index.module.css

```diff
@@ -1,3 +1,10 @@
-.toppMargin {
-  margin-top: 5rem;
```

Comment:
Fikk feil når jeg skulle commite fordi denne var ubrukt i `_index.ts` så derfor den ble flyttet til en egen fil for forsiden i `sider`-mappen.